packages/dd-trace/src/plugins/util/ip_extractor.js

```diff
@@ -41,14 +41,14 @@ function extractIp (config, req) {
   if (config.clientIpHeader) {
     if (!headers) return
 
-    const ip = findFirstIp(headers[config.clientIpHeader])
+    const ip = findFirstIp(headers[config.clientIpHeader.toLowerCase()])
```

Comment:
can we instead convert it to lowercase at the startup instead of on each request ?

Reply:
done.